code/modules/events/disease_outbreak.dm

```diff
@@ -85,11 +85,13 @@
 	announce_when = ADV_ANNOUNCE_DELAY
 	///The disease type we will be spawning
 	var/datum/disease/virus_type
+	///The preset (classic) or generated (advanced) illness name
+	var/illness_type
 	///Disease recipient candidates, passed from the round_event_control object
 	var/list/afflicted = list()
 
 /datum/round_event/disease_outbreak/announce(fake)
-	priority_announce("Confirmed outbreak of level 7 viral biohazard aboard [station_name()]. All personnel must contain the outbreak.", "Biohazard Alert", ANNOUNCER_OUTBREAK7)
+	priority_announce("Confirmed outbreak of level 7 viral biohazard aboard [station_name()]. All personnel must contain the outbreak.", "[illness_type] Alert", ANNOUNCER_OUTBREAK7)
```

Comment:
Is `illness_type` null-safe in this scenario? Sounds like it should be but just wanna make sure since the default is a null (more on that in another review).

Reply:
Yes, I tested it receiving null as illness_type and it's fine. In symptoms.dm we also give it a default of  var/illness = "Unidentified" so it should always have something.